Service/FindOrganizationThroughRepositoriesService.php

```diff
@@ -256,7 +273,7 @@ public function importOrganisation($organisation): ?ObjectEntity
 
         isset($this->io) && $this->io->comment('Checking organisation '.$organisation['login']);
         $synchronization->setMapping($organisationMapping);
-        $synchronization = $this->synchronizationService->synchronize($synchronization, $organisation);
+        $synchronization = $this->synchronizationService->handleSync($synchronization, $organisation);
```

Comment:
Is there a reason to revert to handleSync? The synchronize function should replace handleSync in order to force remapping if the mapping changes.